multimodal-search-pdf/pods/index-comp-text-cc.yml

```diff
@@ -0,0 +1,50 @@
+jtype: CompoundIndexer #this is a indexer for chunks of chunks level, so traversal_path will be one level lower
+components:
+  - jtype: NumpyIndexer
+    with:
+      index_filename: vec.gz
+      metric: cosine
+    metas:
+      name: vecidx  # a customized name
+  - jtype: BinaryPbIndexer
+    with:
+      index_filename: chunk.gz
+    metas:
+      name: docidx
+metas:
+  name: '${{ ENV.indexer_name }}'
+  workspace: $JINA_WORKSPACE/indexed
+requests:
+  use_default: true
+  on:
+    SearchRequest:
```

Comment:
Add docstring for traversal path describing why we use `cc`, `ccm`, describe why we exclude `embedding` and `blob`  in `ExcludeQL`

Reply:
Thanks! @rutujasurve94, I just addressed these comments 😄